Makefile

```diff
@@ -10,4 +10,15 @@ install:
 	# files in /usr/lib
 	install -d $(DESTDIR)$(PREFIX)/lib/gotify-dunst/
 	install main.py $(DESTDIR)$(PREFIX)/lib/gotify-dunst/
-	install gotify-dunst.conf $(DESTDIR)$(PREFIX)/lib/gotify-dunst/
\ No newline at end of file
+	install gotify-dunst.conf $(DESTDIR)$(PREFIX)/lib/gotify-dunst/
+
+	# files in /usr/share
+	install gotify-dunst.desktop $(DESTDIR)$(PREFIX)/share/applications
```

Comment:
Needs an
`install -d $(DESTDIR)$(PREFIX)/share/applications`

Reply:
Required because Arch packages start off with an empty directory. I think other package formats do the same.